### ISSUE TITLE
Make it intuitive for people to switch instances once logged in

### ISSUE THREAD

**Issue by mwc-design** [mwcproject/mwc-qt-wallet#278]
The obvious place to do this would be within the 'account options' dropdown.  Add an option above 'accounts' that says 'instances'

when selected it would bring up a view showing the available instances, someone could click one to change to it.

Will add a mockup in a comment below shortly.

**Comment by cgilliard** [comment]
One issue is that any directory can be an instance so there's no real way to get a list. At least in the current design.

**Comment by mwc-design** [comment]
If it's not possible to list out the directories we can have someone find the directory like we are currently doing.  I was thinking we could scan the mwc-qt-wallet/ directory and exclude things like /tmp/ /context/ /logs/ , but maybe that's not a good solution if someone can choose to put the directory anywhere they want.

**Comment by cgilliard** [comment]
Yeah, I think that's doable. We could probably even do a bit better because
all the instance directories have certain things in them like a wallet.seed
file. We can list all directories in mwc-qt-wallet that have a wallet.seed
file for instance.

**Comment by cgilliard** [comment]
But, we should have a way to find other directories as well (i.e. other
that opens a dialog box to select another directory).
